pkg/controller/registrysyncer/registrysyncer_test.go

```diff
@@ -745,3 +832,38 @@ func TestDockerImageImportedFromTargetingCluster(t *testing.T) {
 		})
 	}
 }
+
+func TestWeird(t *testing.T) {
```

Comment:
?

Reply:
This is to show that fakeClient does not return the expected timestamp.

Reply:
okay but we should probably not merge that :)

Reply:
Removed.